### DISCUSSION TITLE
Suppressing nx output when using run-many

### DISCUSSION THREAD

**Discussion by SirensOfTitan** [nrwl/nx#15198] [General]
I'm trying to run some nx command: `nx run-many lint-all` without all of the nx preambles muddying up the output like:

Before the actual output:
```
 >  NX   Running target lint-all for 2 projects:

[...]
```

After the actual output:
```
 >  NX   Running target lint-all for 2 projects failed

   Failed tasks:
  [...]
```

I want nx to be totally silent when performing these operations so just the concatenation of the various `lint-all` tasks in packages is output.  Is this possible right now?

**Comment by miccou** [comment]
I'd really like to see this too - because running `nx run-many --target=lint --format=compact --output-file=lint-results.txt` doesn't work well - each start of lint will clear the results file so you only get the results for the last project.

It'd be great to skip `output-file=lint-results.txt` and pipe the output to a file, but then the file gets filled with lots of nx output like:
```
 >  NX   Ran target lint for 8 projects (9s)

         With additional flags:
           --quiet=true
           --format=compact

    ✔    7/8 succeeded [0 read from cache]

    ✖    1/8 targets failed, including the following:
    ```

**Reply by mrfelton** [reply]
Similar issue/request as per https://github.com/nrwl/nx/issues/14524
